azure-ts-aks-multicluster/Pulumi.yaml

```diff
@@ -0,0 +1,13 @@
+name: azure-ts-aks-multicluster
+runtime: nodejs
+description: Create multiple Azure Kubernetes Service (AKS) clusters in different regions and with different node counts.
```

Comment:
Nit: the description tends to look better when displayed in the CLI without the period at the end.